### ISSUE TITLE
Broken ARIA references Image List Content Type in CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#15821]
## Describe the defect
Within the edit screens on the Image List content type, there are multiple `aria-describedby` attributes that do not correspond to an ID anywhere on the page.
  - Page Introduction
  - Calls to Action
  - VA Benefits Hubs
  - Related Information
  - Other Categories where people may look for this article

## AC / Expected behavior
- [ ] Determine what the original intention of the ARIA reference was/is.  What additional context was it providing? Do we still need to provide that information in some other way?
- [ ] UX to determine display of additional information if it is determined this is needed
- [ ] Update attribute based on above determination (remove attribute or update it)
- [ ] Accessibility review

## Screenshots
Screenshot of the Page Introduction field error from AMP plugin. The `aria-describedby` attribute is pointing to something with the ID `edit-field-intro-text-limited-html-0--description` but if I search for that ID I can't find it anywhere on the page, it doesn't exist. 

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/6e5a871b-d1f8-4f9a-bcfb-f94eb48a4130)

Screenshot of where the attribute is on the `<div>`
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/74ccfcb4-2faa-4ddc-bb71-a8ec52647a92)

## Accessibility Standard

WCAG version 2.0 A, [Criterion 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

**Comment by laflannery** [comment]
@EWashb (or maybe @jilladams) are you able to tell me who owns Checklist, I can't tell from [this mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/vagov6717/1658961045082/cedcb1c5c42489e47be37733c73890ef890070ae?sender=u10a597890f85398daa3c4805) who should be assigned this ticket

**Comment by jilladams** [comment]
Great question that I also need to ask from a different ticket today. For reference: there are 2 checklist nodes in the whole CMS, both in draft. I suspect DaveC knows the history, but I'll give Erika first right to opine about ownership, before pulling him in. ;)

**Comment by EWashb** [comment]
I found 6 checklists in draft right now and they are all tied to Resources and Support content. I don't have the historical context of the intent of this content type, but I would agree that if they only exist currently within a PW product, that they are a PW product. I am open to conversation if folks feel otherwise. @jilladams @laflannery

**Comment by laflannery** [comment]
I saw [a convo in slack](https://dsva.slack.com/archives/C52CL1PKQ/p1699044675862109?thread_ts=1699029137.297579&cid=C52CL1PKQ) that led me to putting this under PW ownership

**Comment by FranECross** [comment]
@laflannery We don't feel we'll be able to answer these questions/AC (below). Are you able to instead provide recommendations on what should be done?

- Determine what the original intention of the ARIA reference was/is. What additional context was it providing? Do we still need to provide that information in some other way?
-  UX to determine display of additional information if it is determined this is needed

**Comment by qjac** [comment]
Most of the original broken aria-references that were found were fixed in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/22593.

But the Phone number within Contact information, as seen on https://prod.cms.va.gov/node/6954/edit, still errors.

**Comment by dsasser** [comment]
@FranECross FYSA there are only 2 'Image List' nodes, and they are both unpublished. This content type doesn't get used. I'm not pushing back, just passing the info along.

**Comment by FranECross** [comment]
Closing as a no op at this time.
